storm-client/src/jvm/org/apache/storm/trident/Stream.java

```diff
@@ -349,15 +355,15 @@ public TridentState partitionPersist(StateFactory stateFactory, Fields inputFiel
 
     public TridentState partitionPersist(StateSpec stateSpec, Fields inputFields, StateUpdater updater, Fields functionFields) {
         projectionValidation(inputFields);
-        String id = _topology.getUniqueStateId();
-        ProcessorNode n = new ProcessorNode(_topology.getUniqueStreamId(),
-                                            _name,
+        String id = topology.getUniqueStateId();
+        ProcessorNode n = new ProcessorNode(topology.getUniqueStreamId(),
+                name,
```

Comment:
Shouldn't the other params move into this level as well?

Reply:
This is fixed as well, but not displayed outdated.